package.json

```diff
@@ -1,42 +1,38 @@
 {
   "name": "jirafy-sync",
-  "version": "2.0.0",
+  "version": "2.0.1",
   "description": "Sync Jira tickets and version based on a given changelog",
   "main": "dist/index.js",
   "scripts": {
     "build": "ncc build ./index.js",
-    "precommit-msg": "echo 'Pre-commit checks...' && exit 0",
-    "precommit": "npm run format && npm run lint && npm run build && git add dist/",
+    "precommit": "npm run format && npm run build && git add dist/",
     "format-check": "prettier --check **/*.{js,yml,json}",
     "format": "prettier --write **/*.{js,yml,json}",
-    "lint": "eslint ./",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/onxmaps/jirafy-sync.git"
+    "url": "git+https://github.com/onXmaps/jirafy-sync.git"
   },
   "keywords": [
-    "Jira",
-    "Version",
+    "jira",
+    "version",
     "fixVersion",
     "changelog",
     "sync"
   ],
-  "productName": "Jirafy Sync",
-  "author": "Colt Dorsey <colt.dorsey@onxmaps.com",
+  "author": "onXmaps",
   "license": "MIT",
```

Comment:
You can add `private: true` in here to protect it from getting published on npm accidentally. https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private

Reply:
> You can add `private: true` in here to protect it from getting published on npm accidentally. https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private

Given that the jirafy-sync repo is intentionally public, would the "private": "true" suggestion still be a relevant suggestion?